forui/lib/src/theme/style.dart

```diff
@@ -7,8 +7,13 @@ final class FStyle with Diagnosticable {
   /// The border radius.
   final BorderRadius borderRadius;
 
+  /// The text style.
+  final TextStyle textStyle;
+
   /// Creates an [FStyle].
-  FStyle({BorderRadius? borderRadius}) : borderRadius = borderRadius ?? BorderRadius.circular(8);
+  FStyle({BorderRadius? borderRadius, TextStyle? textStyle}):
+    borderRadius = borderRadius ?? BorderRadius.circular(8),
+    textStyle = textStyle ?? const TextStyle(fontSize: 10);
```

Comment:
I personally think it should be stored in `FStyle`. Font should be responsible for only font-related information (not text styling). IMO this is consistent with how the other styles store textstyles if we think of `FStyle` as being the overarching style for the application.

Reply:
Placement of the TextStyle aside, the font size doesn't really do much alone. We need to set the color and probably a list of TextStyles for different situations. I have strong doubts if TextStyle should even be offered in this manner.

Reply:
It could probably make more sense if you want to include constant int for the font size, so we can use it similar to how tailwind defines font sizes.

https://tailwindcss.com/docs/font-size